addons/website/static/tests/tours/edit_menus.js

```diff
@@ -294,6 +294,97 @@ wTourUtils.registerWebsitePreviewTour('edit_menus', {
             this.$anchor[0].dispatchEvent(new window.KeyboardEvent("keydown", { key: "ArrowDown" }));
         },
     },
+    ...wTourUtils.clickOnSave(),
+    // Nest and re-arrange menu items for a newly created menu
+    {
+        content: "open site menu",
+        extra_trigger: "iframe #wrapwrap",
+        trigger: 'button[data-menu-xmlid="website.menu_site"]',
+    },
+    {
+        content: "Click on Edit Menu",
+        trigger: 'a[data-menu-xmlid="website.menu_edit_menu"]',
+    },
+    {
+        content: "Trigger the link dialog (click 'Add Menu Item')",
+        extra_trigger: '.o_website_dialog:visible',
+        trigger: '.modal-body a:eq(0)',
+    },
+    {
+        content: "Fill label input.",
+        extra_trigger: '.o_website_dialog:eq(1):visible',
+        trigger: '.modal-dialog .o_website_dialog input:eq(0)',
+        run: 'text new_menu',
+    },
+    {
+        content: "Fill url input.",
+        trigger: '.modal-dialog .o_website_dialog input:eq(1)',
+        run: 'text #',
+    },
+    {
+        content: "Confirm the new menu entry",
+        trigger: '.modal-footer .btn-primary',
+    },
+    {
+        content: "Trigger the link dialog (click 'Add Menu Item')",
+        extra_trigger: '.o_website_dialog:visible',
+        trigger: '.modal-body a:eq(0)',
+    },
+    {
+        content: "Fill label input.",
+        extra_trigger: '.o_website_dialog:eq(1):visible',
+        trigger: '.modal-dialog .o_website_dialog input:eq(0)',
+        run: 'text new_nested_menu',
+    },
+    {
+        content: "Fill url input.",
+        trigger: '.modal-dialog .o_website_dialog input:eq(1)',
+        run: 'text #',
+    },
+    {
+        content: "Confirm the new menu entry",
+        trigger: '.modal-footer .btn-primary',
+    },
+    {
+        content: "Nest 'new_nested_menu' under 'new_menu'",
+        trigger: ".oe_menu_editor li:contains('new_nested_menu') .fa-bars",
+        run: "drag_and_drop_native .oe_menu_editor li:contains('new_menu') .form-control",
+    },
+    {
+        content: "Check if 'new_nested_menu' is nested under 'new_menu'",
+        trigger: ".oe_menu_editor li:contains('new_menu') > ul > li:contains('new_nested_menu')",
+        run: () => { }, // It's a check.
+    },
+    {
+        content: "Drag 'new_menu' above 'Modnar !!'",
+        trigger: ".oe_menu_editor li:contains('new_menu') .fa-bars",
+        run: "drag_and_drop_native .oe_menu_editor li:contains('Modnar !!') .fa-bars",
+    },
+    {
+        content: "Check if 'new_menu' is above 'Modnar !!'",
+        trigger: ".oe_menu_editor li:contains('new_menu') + li:contains('Modnar !!')",
+        run: () => { }, // It's a check.
+    },
+    {
```

Comment:
```
        content: "Check if 'new_nested_menu' is nested under 'new_menu'",
        trigger: ".oe_menu_editor li:contains('new_menu') > ul > li:contains('new_nested_menu')",
        run: () => { }, // It's a check.
    },
    {
        content: "Drag 'new_menu' above 'Modnar !!'",
        trigger: ".oe_menu_editor li:contains('new_menu') .fa-bars",
        run: "drag_and_drop_native .oe_menu_editor li:contains('Modnar !!') .fa-bars",
    },
    {
        content: "Check if 'new_menu' is above 'Modnar !!'",
        trigger: ".oe_menu_editor li:contains('new_menu') + li:contains('Modnar !!')",
        run: () => { }, // It's a check.
    },
    {
```

You can avoid this 3 steps. No need to check each nesting element separately.
Instead just do,

- Nest menu 1
- Nest menu 2
- Check nesting ( both menu in same step) 
- Drag menu
- Check drop